File: add.py
# Python program to add two numbers

# Function to add two numbers
def add_numbers(num1, num2):
    return num1 + num2

# Taking input from the user
num1 = float(input("Enter the first number: "))
num2 = float(input("Enter the second number: "))

# Calling the add_numbers function and storing the result
result = add_numbers(num1, num2)

# Displaying the result
print(f"The sum of {num1} and {num2} is {result}")

File: add.java
import java.util.Scanner;

public class NumberAddition {
    public static void main(String[] args) {
        // Create a Scanner object for input
        Scanner scanner = new Scanner(System.in);

        // Prompt user for two numbers
        System.out.print("Enter first number: ");
        int num1 = scanner.nextInt();

        System.out.print("Enter second number: ");
        int num2 = scanner.nextInt();

        // Add the two numbers
        int sum = num1 + num2;

        // Display the result
        System.out.println("The sum of " + num1 + " and " + num2 + " is: " + sum);

        // Close the scanner
        scanner.close();
    }
}
